cmd/helm/release_testing_test.go

```diff
@@ -17,9 +17,97 @@ limitations under the License.
 package main
 
 import (
+	"fmt"
+	"regexp"
 	"testing"
+	"time"
+
+	"helm.sh/helm/v3/internal/test"
+	"helm.sh/helm/v3/pkg/release"
+	helmtime "helm.sh/helm/v3/pkg/time"
+)
+
+const (
+	tableLinePattern  = `^Last (Completed|Started):\s+(.+)$`
+	verbPosition      = 1 // in the line patterns
+	timestampPosition = 2 // in the line patterns
 )
 
+type outputFormat struct {
+	linePattern regexp.Regexp
+	checkTime   func(raw string) error
+}
+
+func TestReleaseTesting(t *testing.T) {
+	mockReleases := []*release.Release{
+		createMockRelease(),
+	}
+
+	tableOutput := outputFormat{
+		linePattern: *regexp.MustCompile(tableLinePattern),
+		checkTime: func(raw string) error {
+			_, err := helmtime.Parse(time.ANSIC, raw) // Layout/format must be the one actually used in the command output.
+			return err
+		},
+	}
+
+	tests := []cmdTestCase{
+		{
+			name:   "test without logs",
+			cmd:    "test doge",
```

Comment:
nit: let's not use this trademarked name. How about 
```suggestion
			cmd:    "test mytest",
```
or something else creative that's not a brand trademark, please? :pray:

Reply:
Changed in 66f58ba655cd0f73debc695648d18314fce91f8a! :+1: